JGLUE.py

```diff
@@ -592,7 +584,13 @@ def _info(self) -> ds.DatasetInfo:
             raise ValueError(f"Invalid config name: {self.config.name}")
 
     def __split_generators_marc_ja(self, dl_manager: ds.DownloadManager):
-        file_paths = dl_manager.download_and_extract(_URLS[self.config.name])
+        try:
+            file_paths = dl_manager.download_and_extract(_URLS[self.config.name])
+        except FileNotFoundError as err:
+            logger.warning(err)
+            urls = _URLS[self.config.name]
+            urls.pop("data")  # type: ignore[attr-defined]
```

Comment:
I would be grateful if you could add a comment with the reason for the `.pop`.

Reply:
I have added a comment. Please check it.

As an aside, it seems that JMTEB does not download from https://s3.amazonaws.com/amazon-reviews-pds/tsv/amazon_reviews_multilingual_JP_v1_00.tsv.gz 

https://huggingface.co/datasets/sbintuitions/JMTEB/blob/7e8ad98b751ab42c572ceb4e7723a2885a3d2cfe/sts.py#L13-L67